include/aikido/control/detail/JacobianExecutor-impl.hpp

```diff
@@ -0,0 +1,237 @@
+#include <Eigen/Dense>
+#include <dart/common/Console.hpp>
+#include <dart/dynamics/dynamics.hpp>
+
+#include "aikido/common/memory.hpp"
+#include "aikido/control/KinematicSimulationJointCommandExecutor.hpp"
+#include "aikido/control/util.hpp"
+
+namespace aikido {
+namespace control {
+
+//==============================================================================
+extern template class JacobianExecutor<ExecutorType::VELOCITY>;
+
+extern template class JacobianExecutor<ExecutorType::VELOCITY>;
```

Comment:
Repetition. Should this be `extern template class JacobianExecutor<ExecutorType::EFFORT>;`?